docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md

```diff
@@ -80,3 +80,48 @@ on_worker_boot do
   )
 end
 ```
+
+## Initialization Options
+
+The SDK exposes various initialization options which can be set when registering the DevCycle Client:
+
+```ruby
+# Load the gem
+require 'devcycle-ruby-server-sdk'
+
+# Setup authorization
+options = DevCycle::Options.new(enable_edge_db: true, enable_cloud_bucketing: true)
```

Comment:
should the basic example of how to use options really include a bunch of options that won't mean anything to the reader at this point?

Reply:
swapped the basic example to be what we want, enable realtime updates, make config poll longer